src/components/Header/NetworkCard.tsx

```diff
@@ -183,24 +188,37 @@ export default function NetworkCard() {
 
   if (L2_CHAIN_IDS.includes(chainId)) {
     const info = L2_INFO[chainId]
+    let bridgeText, explorerText
+    switch (chainId) {
+      case SupportedChainId.ARBITRUM_ONE:
+      case SupportedChainId.ARBITRUM_RINKEBY:
+        bridgeText = `${NETWORK_LABELS[chainId]} Bridge`
```

Comment:
can you wrap those with `<Trans>` unless it's the product name? Can also use `t` macro if you need a string (although be aware that there's a known bug where `t` doesn't update on locale change)